assets/sass/components/table.global.scss

```diff
@@ -0,0 +1,104 @@
+@use "../_func.scss" as *;
+
+// #region Default table
+table {
+  --border-width: 2px;
+  --inner-border-width: 2px;
+  --hover-background: var(--colour-light);
```

Comment:
Think we need an alternative table hover background for dark mode. There isn't much contract between the background and text colour of the cells

Reply:
Updated this and moved the change to a different file which makes more sense.